### ISSUE TITLE
Client HTML processor loses border style during processing (in IE)

### ISSUE THREAD

**Issue by churkin** [DevExpress/testcafe-hammerhead#240]
This test will fail:

``` javascript
test('The HTML processor loses border style during processing', function () {
    var html = '<table style="border-style:None;"></table>';

    strictEqual(htmlUtils.processHtml(html), html);
});
```

IE browsers remove this style from the markup.

**Comment by churkin** [comment]
Will be fixed after #242.

**Comment by LavrovArtem** [comment]
fixed
